velox/docs/functions/spark/math.rst

```diff
@@ -108,11 +108,20 @@ Mathematical Functions
 
 .. spark:function:: rand() -> double
 
-    Returns a random value with independent and identically distributed uniformly distributed values in [0, 1). ::
+    Returns a random value with independent and identically distributed uniformly distributed values in [0, 1).
+    Its alias is ``random``.
 
         SELECT rand(); -- 0.9629742951434543
-        SELECT rand(0); -- 0.7604953758285915
-        SELECT rand(null); -- 0.7604953758285915
+
+.. spark:function:: rand(seed) -> double
+
+    Returns a random value like the above rand function. Its alias is ``random``.
```

Comment:
Returns a random value with independent and identically distributed uniformly distributed values in [0, 1) using specified seed. NULL seed is identical to zero seed.